sdk/storage/azure-storage-file-datalake/tests/test_file.py

```diff
@@ -1002,12 +1183,15 @@ def test_rename_file_with_file_sas(self, datalake_storage_account_name, datalake
         new_client = file_client.rename_file(file_client.file_system_name+'/'+'newname'+'?'+new_token)
 
         data = new_client.download_file().readall()
-        self.assertEqual(data, data_bytes)
-        self.assertEqual(new_client.path_name, "newname")
+        assert data == data_bytes
+        assert new_client.path_name == "newname"
 
     @pytest.mark.skip(reason="Service bug, requires further investigation.")
```

Comment:
Did you try running this? We should probably "investigate further" :)

Reply:
Working now, sweet beans 😄